index.html

```diff
@@ -38,5 +38,85 @@ <h2 class="ocupacion">I'am a Software Developer</h2>
 <section class="ornament">
 </section>
 
+<section class="middle-section">
+
+<div class="my-recent">
+  <p class="recent-title">My Recent Works</p>
+<img class="recent-img" src="pictures/Img Placeholder.svg" alt="">
+<p class="recent-title2 color-dark">Multi-Post Stories</p>
+<p class=" contend color-dark">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+<ul class="color-dark">
+  <li>CSS</li>
+  <li>html</li>
+  <li>boodstrap</li>
+  <li>Ruby</li>
+</ul>
+<a class="proyect-recent see-style" href="#">See Project</a>
+</div>
+
+<div class="works works1 bak-image work-container">
+  <p class="works-title">Profesional Art Printing Data</p>
```

Comment:
- [ ]  Each project title should use header tag `hx`  To give a good structure to your HTML, with good cemetic elements, you should give `hx `tag to each project's title. This will increase your project accessibility.  I have mentioned it here, but  remember to fix it to each of your project.